### PR TITLE
ref: set testdata-db DS for panel plugin

### LINKED ISSUE #1112: Feat: Replace random walk in provisioned panel dashboard
### Which areas does this feature request relate to

- [x] Create Plugin
- [ ] Sign Plugin
- [ ] Plugin E2E
- [ ] Plugin Meta Extractor
- [ ] Documentation

### Problem

Feedback suggests developers that are new to Grafana plugins or dataframes find the random walk datasource quite confusing as it's not explicit what a dataframe looks like, it changes on every refresh, and it is limited when it comes to controlling what is sent to a panel plugin.

### Solution

Replace the random walk datasource that is currently set in the panel plugins provisioned dashboard with either:

- test data datasource set to "raw frames"
- replace test data datasource with [static datasource](https://grafana.com/grafana/plugins/marcusolsson-static-datasource/)

Either of these options should solve this issue and help developers to more easily build their panel plugin.

### Alternatives

_No response_

### Additional context

_No response_

### Are you interested in contributing the solution?

- [ ] Yes
- [ ] No

### PR DESCRIPTION
**Pull request by s4kh**
**What this PR does / why we need it**:

Configures [TestData DB](https://grafana.com/docs/grafana/latest/datasources/testdata/) for provisioned panel dashboard .

**Which issue(s) this PR fixes**:

Fixes https://github.com/grafana/plugin-tools/issues/1112
Fixes https://github.com/grafana/grafana-community-team/issues/243

Screenshot from 2024-10-31 10-22-01

### CODE REVIEW

**Comment by s4kh** [comment]
@jackw For the dashboard, we also have https://github.com/grafana/plugin-tools/pull/1282/files#diff-f5a693e0319d1b6e0c9ae180ebcb1aa6241150e9a48377120ec138f303ea91b2R81 config for same panel but `scenarioId`:`no_data_points`. Should we change this as well?

**Comment by s4kh** [comment]
@tolzhabayev @jackw 
One weird thing is if I change the scenario to something else and comeback again at `Raw Frame` the data is lost, refreshing the page populates the data. It seems the expected behaviour.

File: packages/create-plugin/templates/panel/provisioning/dashboards/dashboard.json [hunk 1]
```diff
@@ -22,36 +22,6 @@
   "links": [],
   "liveNow": false,
   "panels": [
-    {
```
Comment (s4kh) [inline packages/create-plugin/templates/panel/provisioning/dashboards/dashboard.json:25]: I don't think we need to see the same plugin with different scenario in the dashboard. It is confusing to see the panels one with circle other one without circle(due to no data).

**Review by mckn — APPROVED** [review]
Nice work with this! LGTM

**Review by josmperez — APPROVED** [review]
Looks good overall, nice additions. I've made a commit with some suggestions mostly for Grafana style and clarity. Please take a look and approve if they make sense to you.

**Review by xnyo — APPROVED** [review]
LGTM! :rocket:

**Pull request merged**